src/lib/Translation/Extractor/OnlineEditorCustomAttributesExtractor.php

```diff
@@ -55,7 +52,7 @@ public function extract(): MessageCatalogue
         $catalogue->add($this->createMessage(self::CLASS_LABEL_MESSAGE_ID, 'Class'));
 
         foreach ($this->siteAccessList as $scope) {
-            if (!$this->configResolver->hasParameter(RichText::ATTRIBUTES_SA_SETTINGS_ID)) {
+            if (!$this->configResolver->hasParameter(RichText::ATTRIBUTES_SA_SETTINGS_ID, null, $scope)) {
```

Comment:
Here I'm not sure if extraction should be done in some other way, as without this change none of custom attributes are extracted (as in yaml example in docs https://doc.ibexa.co/en/latest/content_management/rich_text/extend_online_editor/#custom-data-attributes they're in `admin_group` and default scope for `hasParameter` is `site`, at least from my little investigation.
So someone from backend has to verify it 100%. :)

Reply:
Make sense for me.